posts/2022-08-24-open-liberty-with-cargo-tracker.adoc

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Try out Eclipse Cargo Tracker, an enterprise-level Jakarta EE application, with Open Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ocseal
+author_github: https://github.com/ocseal
+seo-title: Try out Eclipse Cargo Tracker, an enterprise-level Jakarta EE application, with Open Liberty - OpenLiberty.io
+seo-description: Experience Open Liberty in an industrial environment by using a Liberty server to run Eclipse Cargo Tracker, an open-source application built to simulate real-world implementations of Jakarta EE.
+blog_description: "Experience Open Liberty in an industrial environment by using a Liberty server to run Eclipse Cargo Tracker, an open-source application built to simulate real-world implementations of Jakarta EE."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Try out Eclipse Cargo Tracker, an enterprise-level Jakarta EE application, with Open Liberty
+Chanun Ong <https://github.com/ocseal>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+As one of the top Java framework choices for enterprise-level applications, Jakarta EE can be found at the center of projects all over the globe. The https://eclipse-ee4j.github.io/cargotracker/[Eclipse Cargo Tracker] open-source application was designed to replicate a complex enterprise application by utilizing core Jakarta EE features with professional development standards. Now, you can get a taste of Open Liberty in an industrial environment by running Cargo Tracker with an Open Liberty server all on your own computer.
+
+== The Cargo Tracker application
+
+Cargo Tracker simulates an industrially-developed freight tracking web application built with Jakarta EE 8 and domain-driven design principles. The application uses multiple key Jakarta EE features, including Faces, CDI, enterprise beans, bean validation, REST, and JSON binding. Persistence is also implemented along with batching and Jakarta messaging. 
```

Comment:
Hmm... lowercase vs. uppercase , proper vs. common noun.... Since this is a Jakarta-centric sample I think we're highlighting the Jakarta technologies: https://jakarta.ee/specifications/  and so it should be more like: 

"The application uses multiple key Jakarta EE features, including Faces, CDI, Enterprise Beans, Bean Validation, RESTful Web Services, and JSON Binding.  Persistence is also used along with Batch and Messaging."

Since we lead into it mentioning Jakarta I think you can omit it next to each technology.